### PR TITLE
Fix Google IP

### PR DESCRIPTION
**Pull request by gmist**

### CODE REVIEW

**Comment by lipis** [comment]
Why did they disable it?! :( this might happen again I guess.. what about checking for www.google.com? Will take more time for the DNS resolution.. but might be more future proof

**Comment by lipis** [comment]
:dancer: you did it..

File: run.py [hunk 1]
```diff
@@ -124,7 +124,7 @@
 # Other global variables
 ###############################################################################
 CORE_VERSION_URL = 'https://gae-init.appspot.com/_s/version/'
-INERNET_TEST_URL = 'http://74.125.228.100'
+INERNET_TEST_URL = 'http://google.com'
```
Comment (lipis) [inline run.py]: as this is a redirect.. maybe it's better to use the https://www.google.com or it doesn't matter?

File: run.py [hunk 2]
```diff
@@ -124,7 +124,7 @@
 # Other global variables
 ###############################################################################
 CORE_VERSION_URL = 'https://gae-init.appspot.com/_s/version/'
-INERNET_TEST_URL = 'http://74.125.228.100'
+INERNET_TEST_URL = 'http://www.google.com'
```
Comment (lipis) [inline run.py]: `httpS` :D

Comment (gmist) [inline run.py]: I do not think that this is important, but so be it.

**Comment by lipis** [comment]
now you can see where I'm :) and probably that's why there is not much going on.. :( https://gae-init.appspot.com/stats/ stupid Google :)

**Comment by gmist** [comment]
Hot place for holidays

**Comment by joenorton** [comment]
:+1:  Ah, here I was thinking I was having wifi issues!

**Pull request merged**